api/src/main/resources/emails/welcomeemail/WelcomeEmail.html

```diff
@@ -0,0 +1,125 @@
+      display: inline-block;
+      margin-bottom: 0;
+    }
+    .user-info {
+      font-weight: bold;
+      display: inline-block;
+    }
+    .card {
+      height: 144px;
+      width: 540px;
+      border-radius: 4px;
+      background-color: #FFFFFF;
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet-img {
+      height: 31px;
+      width: 31px;
+    }
+
+    .card-text {
+      width: 194px;
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+    }
+
+    .step-img {
+      height: 120px;
+      width: 250px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://accounts.google.com/signin/oauth/oauthchooseaccount?hd=fake-research-aou.org&client_id=602460048110-5uk3vds3igc9qo0luevroc2uc3okgbkt.apps.googleusercontent.com&as=1ivZqDJMGPEMvWasjAquSQ&destination=http%3A%2F%2Flocalhost%3A4200&approval_state=!ChRqd2tjYnk1b040d3VUMzNpZXBpQRIfczlfc0R4d2dERHdUQU9FaERQUmpBazFLYU13VFBSWQ%E2%88%99AB8iHBUAAAAAWxgpw3y6ICN3aJ_etNsSGfEmZbFNYV2d&xsrfsig=AHgIfE85mPI6c5DfCze82tpt7pbw8-XBRg&flowName=GeneralOAuthFlow">
+    <img class="bullet-img" src="assets/Bullet1@2x.png">
+    <div class="card-text"><div class="link">Sign into google</div> with your new account</div>
+    <img class="step-img" src="assets/Step1@2x.png">
+  </a>
+  <div class="card">
+    <img class="bullet-img" src="assets/Bullet2@2x.png">
+    <div class="card-text">Google will then prompt you to change your password</div>
+    <img class="step-img" src="assets/Step2@2x.png">
+  </div>
+  <a class="card" href="https://myaccount.google.com/security">
+    <img class="bullet-img" src="assets/Bullet3@2x.png">
+    <div class="card-text">Using the Researcher Workbench will require you to <div class="link">set up 2-Step Verification</div></div>
+    <img class="step-img" src="assets/Step3@2x.png">
```

Comment:
Are we really embedding full screenshots of the Google login UI in our email? This seems really primitive to me.. We're also subject to breaking on account of arbitrary changes in Google's UI, including whether they run experiments across users.

Maybe this is OK as a short-term solution, but I think we should have some kind of plan to get rid of this.

Reply:
Yeah, if we can route directly to 2SV, this becomes way less important. Will sync with Lou tomorrow.

Reply:
Agreed that if we can point the user to the right places clearly and succinctly, we should do that. Not sure how feasible that is given the layout of the current 2SV google page.

Reply:
So, now that we can route directly to 2sv and recovery email, it is probably possible to do and be clear without images. I synced up with @lstrano-broad this morning, and he mentioned wanting to avoid having the email be just text from a design standpoint though. The dependency on google's UI bothers me some though, that's true. He suggested we check in with @fknabe to get a product decision here.